cscs-checks/apps/gromacs/gromacs_check.py

```diff
@@ -76,10 +76,10 @@ def __init__(self, **kwargs):
         self.tags |= {'production'}
         self.reference = {
             'dom:gpu': {
-                'perf': (15.7, -0.15, None)
+                'perf': (32.05, -0.05, None)
```

Comment:
I would round to `32.0` so it is easier to parse. What do you think?

Reply:
I would say either `32.0` or `32.1` depending on the real median value.